src/skirmishbunker/bunkerBody.py

```diff
@@ -35,8 +36,8 @@ def make_interior_rectangle(self):
 
     self.interior_rectangle = (
         cq.Workplane("XY")
-        .box(self.int_length, self.int_width, self.height-self.wall_width)
```

Comment:
This actually serves a bit of a purpose. The interior rectangle creates the space for the room inside.
This should be  minus wall_width when floor_thickness is None. and when floor_thickness is set that should be minus'd from the height instead.

suggested when floor_thickness is set:
.box(self.int_length, self.int_width, self.height-self.floor_thickness)